datastore/__init__.py

```diff
@@ -154,10 +154,10 @@ async def create_picture_set(
                 f"User not found based on the given id: {user_id}"
             )
 
-        picture_set = data_picture_set.build_picture_set(user_id, nb_pictures)
+        picture_set_metadata = data_picture_set.build_picture_set(user_id, nb_pictures)
         picture_set_id = picture.new_picture_set(
             cursor=cursor,
-            picture_set=picture_set,
+            picture_set=picture_set_metadata,
```

Comment:
Should we rename `build_picture_set` altogether? Asking because if `picture_set` was not descriptive enough, then maybe the function as well? And the param name in `new_picture_set` as well?

Reply:
I'm all for it. Should we do it in this issue?

Reply:
done

Reply:
What about the parameter name?

Reply:
done